.rubocop.yml

```
@@ -0,0 +1,120 @@
+---
```

We'd need to add `require: rubocop-rails` in order to enable the Rails cops:

```suggestion
---
require: rubocop-rails
```